packages/core/src/utils/coin-formatter.ts

```diff
@@ -0,0 +1,61 @@
+import BigNumber from 'bignumber.js'
+import { unitToCurrency, assets } from '@liquality/cryptoassets'
+import { Asset } from '@liquality/cryptoassets/dist/src/types'
```

Comment:
why isn't this type exported directly?

Reply:
This should be imported from the liquality types package which I did not exist until recently.

Reply:
I don't see `Asset` as an export of `@liquility/types`

Reply:
If it does not, we then need to talk to the team working on the CAL to add it to the types repo or we can add it ourselves.